Formula/libdv.rb

```diff
@@ -19,6 +19,14 @@ class Libdv < Formula
 
   depends_on "popt"
 
+  on_macos do
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
```

Comment:
Probably needs pkg-config to avoid error:
```suggestion
    depends_on "libtool" => :build
    depends_on "pkg-config" => :build
```

Reply:
Wow, thanks for catching this!  That was the problem.